vhd/packer/install-dependencies.sh

```diff
@@ -296,9 +296,10 @@ done
 
 NVIDIA_DEVICE_PLUGIN_VERSIONS="
 1.11
```

Comment:
do we still need `1.11`?

Reply:
No, not if 1.0.0-beta.6 replaced it. 

We don't need to keep anything that's not referenced on the VHD (AKS Engine used to do very lazy garbage collection here when the image was shared with AKS, but that's not necessary now).

Reply:
thanks! removed